src/au3wrap/internal/trackcolor.cpp

```diff
@@ -21,6 +21,10 @@ TrackColor& TrackColor::Get(const Track* track)
 TrackColor::TrackColor(Track& track)
     : mTrack{track.shared_from_this()}
 {
+    if (!projectSceneConfiguration()) {
+        return;
+    }
```

Comment:
I think better add mock, like this https://github.com/musescore/MuseScore/blob/master/src/engraving/tests/environment.cpp 

in the future there will be more and more places like this